audit_log.py

```diff
@@ -0,0 +1,170 @@
+import socket
+import ssl
+import sys
+import argparse
+import ConfigParser
+import requests
+from jinja2 import Template
+import os
+import json
+import time
+import logging
+import logging.handlers
+from cb_defense_syslog import *
+import traceback
+import hashlib
+import fcntl
+
+
+#Get Audit Logs will return response
+def get_audit_logs(url, api_key_query, api_connector_id_query, ssl_verify, logger, proxies=None):
+    headers = {'X-Auth-Token': "{0}/{1}".format(api_key_query, api_connector_id_query)}
+    try:
+        response = requests.get("{0}/integrationServices/v3/auditlogs".format(url),
+                                headers=headers,
+                                timeout=15, proxies=proxies)
+
+        if response.status_code != 200:
+            logger.error("Could not retrieve audit logs: {0}".format(response.status_code))
+            return False
```

Comment:
It's kind of strange to return False in some cases but None/Response in the other. I would recommend making all False -> None

Reply:
Made these changes. Will incorporate these in the next push.